test/schema-actions.js

```diff
@@ -9,7 +9,7 @@ const { getSchemaDir } = require('./utils');
 metatests.test('must load Actions', test => {
 
   const testMethod = (test, method, ctx, expected) => {
-    method({}, ctx, {}, (err, actual) => {
+    method({}, ctx, {}).then((actual, err) => {
```

Comment:
`Promise.prototype.then()` passes only one argument to the `onFulfilled` function (first argument to `then()`). You should add a second argument (`onRejected` function) to `then()` here, to handle error.